heasarc/sciserver_heasoft/Dockerfile

```diff
@@ -100,7 +100,7 @@ RUN cd /home/${sciserver_user}/heasoft-${HEASOFT_VERSION}/BUILD_DIR/ \
 # Tweak Xspec settings for a no-X11 environment
 # add xspec and xstar model data from the data location in ~/workspace/headata/FTP
 COPY Xspec.init .
-RUN echo -e "setplot splashpage off\ncpd /GIF\n" >> /opt/heasoft/spectral/scripts/global_customize.tcl \
+RUN echo "setplot splashpage off\ncpd /GIF\n" >> /opt/heasoft/spectral/scripts/global_customize.tcl \
```

Comment:
I don't think this is correct. See https://unix.stackexchange.com/questions/189787/difference-between-echo-and-echo-e

Why did you make this change?

Reply:
The first one was print '-e' along with the other text, causing xspec to misbehave sometimes.

Reply:
This produces the expected output. I think it does because the SHELL command a few lines before: `SHELL ["/home/idies/miniconda3/bin/conda", "run", "-n", "heasoft", "--no-capture-output", "/bin/bash", "-c"]
`

Reply:
It's hard to believe it's working. GNU `/bin/echo` and bash's built-in `echo` command both require the "-e" switch for this, so I guess it's using a different `echo` command then. (Does conda implements it's own `echo`?!) If so, it's a very non-standard `echo` to be clear, and I wouldn't trust it. I still recommend changing `echo` to `printf` or `/bin/echo -e`.
```
% /bin/echo -e 'foo\nbar'
foo
bar
% /bin/echo 'foo\nbar'
foo\nbar
% bash
bash-4.1$ echo -e 'foo\nbar'
foo
bar
bash-4.1$ echo 'foo\nbar'
foo\nbar
```

Reply:
Thanks @esabol.
I actually I don't know how it is producing the correct output. To remove ambiguity, I replaced `echo` with `printf` in
2fcdd93e421f99a347c5260355b9c5b3cd09a545.